core/search/min_match_disjunction.hpp

```diff
@@ -159,18 +159,12 @@ class min_match_disjunction : public doc_iterator,
       return doc_.value;
     }
 
-    /* execute seek for all lead iterators and
-     * move one to head if it doesn't hit the target */
+    // execute seek for all lead iterators and
+    // move one to head if it doesn't hit the target
     for (auto it = lead(), end = heap_.end(); it != end;) {
       assert(*it < itrs_.size());
       const auto doc = itrs_[*it]->seek(target);
 
-      if (doc == target) {
```

Comment:
This code change about check target != eof?

Reply:
Why not check it once?

Reply:
no, this checks if doc is equal to target after seek(...). The check was moved.